domains/certificates/Query.API/API/Query.API/Projections/CertificatesByOwnerProjection.cs

```diff
@@ -1,119 +1,186 @@
+
+        view.Certificates[productionCertificateIssued.CertificateId].Status = CertificateStatus.Issued;
+        views.Add(view);
     }
 
-    public void Apply(ProductionCertificateCreated @event, CertificatesByOwnerView view)
+    private static void CreateCertificatesByOwnerView(IDocumentOperations operations,
+        ProductionCertificateCreated productionCertificateCreated)
     {
-        view.Owner = @event.MeteringPointOwner;
-        view.Certificates[@event.CertificateId] = new CertificateView
+        var view = operations.Load<CertificatesByOwnerView>(productionCertificateCreated.MeteringPointOwner);
+        var certificateView = new CertificateView
         {
-            CertificateId = @event.CertificateId,
-            DateFrom = @event.Period.DateFrom,
-            DateTo = @event.Period.DateTo,
-            Quantity = @event.ShieldedQuantity.Value,
-            GSRN = @event.ShieldedGSRN.Value,
-            GridArea = @event.GridArea,
-            TechCode = @event.Technology.TechCode,
-            FuelCode = @event.Technology.FuelCode,
+            CertificateId = productionCertificateCreated.CertificateId,
+            DateFrom = productionCertificateCreated.Period.DateFrom,
+            DateTo = productionCertificateCreated.Period.DateTo,
+            Quantity = productionCertificateCreated.ShieldedQuantity.Value,
+            GSRN = productionCertificateCreated.ShieldedGSRN.Value,
+            GridArea = productionCertificateCreated.GridArea,
+            TechCode = productionCertificateCreated.Technology.TechCode,
+            FuelCode = productionCertificateCreated.Technology.FuelCode,
             Status = CertificateStatus.Creating
         };
-    }
 
-    public void Apply(ProductionCertificateIssued @event, CertificatesByOwnerView view)
-        => view.Certificates[@event.CertificateId].Status = CertificateStatus.Issued;
+        if (view == null)
+        {
+            view = new CertificatesByOwnerView
+            {
+                Owner = productionCertificateCreated.MeteringPointOwner,
+                Certificates =
+                {
+                    [productionCertificateCreated.CertificateId] = certificateView
+                }
+            };
+        }
+        else
+        {
+            view.Certificates.Add(productionCertificateCreated.CertificateId, certificateView);
+        }
 
-    public void Apply(ProductionCertificateRejected @event, CertificatesByOwnerView view)
-        => view.Certificates[@event.CertificateId].Status = CertificateStatus.Rejected;
-}
+        views.Add(view);
+    }
 
-public class CertificatesByOwnerView
-{
-    [Identity]
-    public string Owner { get; set; } = "";
+    private static void ApplyProductionCertificateTransferred(IDocumentOperations operations,
+        ProductionCertificateTransferred productionCertificateTransferred)
+    {
+        var source = operations.Load<CertificatesByOwnerView>(productionCertificateTransferred.Source)
```

Comment:
There is no attempt to get the view from `views` here. That works for now as the the Transferred event will be the only event in the transaction. But it is different compared to how Issued and Rejected events are applied. Would it make sense to have the same logic wrt. `views` for the Apply-methods for Issued, Rejected and Transferred?